src/ui/components/UserAvatar/index.js

```diff
@@ -7,20 +7,33 @@ import type { UserType } from 'amo/reducers/users';
 
 
 type Props = {|
+  altText?: string | null,
   className?: string,
-  user?: UserType,
+  preview?: string | null,
+  user: UserType | null,
 |};
 
-const UserAvatar = ({ className, user }: Props) => {
+const UserAvatar = ({ altText, className, preview, user }: Props) => {
   const _className = makeClassName('UserAvatar', className);
 
+  let image = preview && preview.length ? preview : null;
+
+  if (user && !image) {
+    image = user.picture_type && user.picture_type.length && user.picture_url
```

Comment:
I don't know, that was there 🤷‍♂️ I think that's just the way of determining whether there is a profile picture for a given user or not.

Reply:
Maybe it's worth investigating further, or maybe @eviljeff knows? I don't love having mysterious code that we don't fully understand in our code base.

Reply:
`picture_type` is the mime type of the image [[docs](http://addons-server.readthedocs.io/en/latest/topics/api/accounts.html#account)]
Previously `picture_url` would be a url to a default avatar if there wasn't a user provided picture, so you had to test `picture_type` - which would be `null`/empty if there wasn't a user provided picture - to check if it was set or not.
Since mozilla/addons-server#7679 `picture_url` has been `null` when not set so I don't believe you need that check.

Reply:
Yup, let's forget about `picture_type` then.